examples/images/voc_sift_fisher.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+set -e
+
+#Set environment variables
+: ${KEYSTONE_MEM:=4g}
+export KEYSTONE_MEM
+
+KEYSTONE_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"/../..
+: ${HADOOP_EXAMPLE_DATA_DIR:=/example_data}
+
+
+#Get the data and copy to HDFS if necessary.
+if [ ! hadoop fs -test -e $HADOOP_EXAMPLE_DATA_DIR/VOCtrainval_06-Nov-2007.tar ]; then
+    wget -O $TMPDIR/VOCtrainval_06-Nov-2007.tar https://s3-us-west-2.amazonaws.com/voc-data/VOCtrainval_06-Nov-2007.tar
+    hadoop fs -copyFromLocal VOCtrainval_06-Nov-2007.tar $HADOOP_EXAMPLE_DATA_DIR
```

Comment:
We could also have an ImageNet thing since that's public, such a script would look like this though I hesitate to include it because to run in a reasonable amount of time you need n>10 machines.